### PR TITLE
Disable Nether Roof Interaction

### PR DESCRIPTION
**Pull request by crafter23456**
I cant break the roof in creative or like that, seems robust, but i can place but not break on top of it and the message appears, yeah because its coded like it should do that. cause of my other PRs, the readme order will change anyways so i pasted it just on the bottom of the flamepaper patches

# Checklist:

- [x] I have reviewed my code thoroughly.
- [x] I have tested my code.
- [x] My changes generate no new warnings

### CODE REVIEW

File: NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java [hunk 1]
```diff
@@ -404,4 +405,10 @@ private static void enableImprovedHitReg() {
     private static void disableDisconnectSpam() {
         disableDisconnectSpam = getBoolean("settings.disable-disconnect-spam", false);
     }
+
+    public static boolean disableNetherroofDestroying;
```
Comment (Elierrr) [inline NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java:409]: Incorrect name, the config disabled placing on nether roof not destroying it

File: NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java [hunk 2]
```diff
@@ -143,6 +143,7 @@ static void loadComments() {
         c.addComment("settings.commands.enable-help-command", "Toggles the /help command");
         c.addComment("settings.use-improved-hitreg", "Enables the usage of an improved hitreg based on lag compensation and small other details.");
         c.addComment("settings.disable-disconnect-spam", "Disables that players can be kicked because of disconnect.spam.");
+        c.addComment("settings.disable-nether-roof-destroying", "Disables destorying the nether roof.");
```
Comment (Elierrr) [inline NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java:146]: Update comment

**Review by Elierrr — CHANGES_REQUESTED** [review]

**Comment by Elierrr** [comment]
I don’t really see the point of this since it can very easily be done with a plugin

File: NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java [hunk 3]
```diff
@@ -404,4 +405,10 @@ private static void enableImprovedHitReg() {
     private static void disableDisconnectSpam() {
         disableDisconnectSpam = getBoolean("settings.disable-disconnect-spam", false);
     }
+
+    public static boolean enableNetherroofDestroying;
```
Comment (Elierrr) [inline NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java:409]: It should be `disableNetherroofInteraction`

**Review by Elierrr — CHANGES_REQUESTED** [review]

**Review by Elierrr — APPROVED** [review]
This PR can be merged, however I don’t believe this is necessary since it can easily be done with a plugin

**Comment by Lucaskyy** [comment]
I agree with Elier, plugins should handle this. NachoSpigot is for performance, not features plugins already do. Thanks anyway though!

**Pull request closed without merging**